app/assets/stylesheets/forms/_inputs.scss

```diff
@@ -109,7 +109,7 @@ input[type=radio] {
 			@media (-webkit-min-device-pixel-ratio:0) {
 			    outline-color: -webkit-focus-ring-color;
 			    outline-style: auto;
-			    outline-width: 5px;
+			    outline-width: .313rem;
```

Comment:
i'd be careful about using `.313rem` vs 5px. that's not the same value / proportion. 

i'd also just be very cautious about using magic numbers like .313. not easy to work with.
`percentage(5/16) em` is the way to go.

not only is that way more legible, it will also give you the appropriate precision. 
would strongly recommend doing any place where the value is not obvious.

Reply:
actually, re: `percentage(5/16)`

that can be cleaned up. what you really wanna do is declare another variable, then use that in the calculations.

```$root-font-val: 16

Reply:
going with this:
```
@function rem($numPx) {
	@return ($numPx / $root-font-size) * 1rem;
}
```